docs/getting_started.md

```diff
@@ -203,6 +203,136 @@ See these resources for more information on clustering:
  - [Quantifying clustering behavior in Orchestrating Single Cell Analysis](https://bioconductor.org/books/release/OSCA.advanced/clustering-redux.html#quantifying-clustering-behavior)
 
 
+### Cell type annotation
+
+Processed objects may contain cell type annotations and associated metadata from one or more of the following sources.
+
+- Submitter-provided annotations
+  - Note that these are only present for a subset of libraries.
```

Comment:
```suggestion
- Submitter-provided annotations (note that these are only present for a subset of libraries). 
```
I think you could just put this on the same line and don't need the nested bullet.